### ISSUE TITLE
Wrong face orientation when exporting assemblies to VRML

### ISSUE THREAD

**Issue by mojca** [CadQuery/cadquery#1158]
I'm not yet "proficient in interpreting VRML files", but it seems that the assemblies exported into VRML end up with wrong face orientation.

I first reported the issue to KiCad:
* https://gitlab.com/kicad/code/kicad/-/issues/12400

and prepared the files with a minimum example here:
* https://github.com/mojca/vrml-backface-culling-issue

```python
import cadquery as cq

box = cq.Workplane().box(2.0, 4.0, 1.0)
assy = cq.Assembly(box, loc=cq.Location((0,0,0)), color=cq.Color("black"))

cq.exporters.export(box, 'box-ok.wrl', cq.exporters.ExportTypes.VRML)
assy.save('box-has-back-face-culling-issues.wrl', exportType='VRML')
```

@jmwright

**Comment by adam-urbanczyk** [comment]
Currently VRML export is based on VTK and it uses VTK-based normal calculation. In 7.6 it is possible to use the normals from OCCT. This line would have to be modified.

https://github.com/CadQuery/cadquery/blob/259cc7b4e47cf8da695085ab507963b57c60e046/cadquery/occ_impl/assembly.py#L194

So long story short - let's wait till 7.6 is merged and try to fix it there.

**Comment by mojca** [comment]
If I understand correctly: someone needs to finalize support for Open CASCADE version 7.6.0 in CadQuery and then you'll be able to test an alternative approach to export to VRML and see if it works better? (It's not totally unambiguous what version 7.6 refers to.) Thank you for checking this out.

**Comment by adam-urbanczyk** [comment]
Indeed, [OCCT/OCP] 7.6. See: https://github.com/CadQuery/cadquery/pull/1156

**Comment by adam-urbanczyk** [comment]
@mojca could you check if #1167 solves this issue?

**Comment by mojca** [comment]
What is the best way to install CadQuery in a proper way, so that I will be able to test that change? (I would prefer to stick with `pip` rather than `conda`, if possible. Or I can manually edit some files if needed.)

**Comment by adam-urbanczyk** [comment]
This or clone the repo and use `pip install -e .`. You might want to ignore the deps if you already installed them.

**Comment by mojca** [comment]
Thank you, that seems to work fine (both the installation and rendering a complex model).

**Comment by mojca** [comment]
This leaves two remaining "issues" related to KiCad:
* it would be handy to have an additional export parameter for scale or base unit `assy.save('model.wrl', exportType='VRML', scale=1/2.54)`
* KiCad developers complain about the VRML output format being heavily unoptimized, see https://gitlab.com/kicad/code/kicad/-/issues/12400

But those are pretty unrelated and I'll probably open a new issue for those.

**Comment by adam-urbanczyk** [comment]
Ok, open new issues @mojca, but please use more quantitative language than "something is heavily unoptimized".

**Comment by adam-urbanczyk** [comment]
Solved by #1167
